website/docs/index.html.markdown

```diff
@@ -11,6 +11,8 @@ The Azure DevOps provider can be used to configure Azure DevOps project in [Micr
 
 Use the navigation to the left to read about the available resources.
 
+Interested in the provider's latest features, or want to make sure you're up to date? Check out the [changelog](https://github.com/microsoft/terraform-provider-azuredevops/blob/master/CHANGELOG.md) for version information and release notes.
```

Comment:
@PatOShea @EliiseS  The link to the `CHANGELOG.md` file still points to the former Git repository at Microsoft. Was this intended?

Reply:
Woops! I missed that! Not intended 😂

Reply:
I'll fix it

Reply:
Fixed in #75